kits/home-maintenance-triage/apps/src/app/globals.css

```
@@ -0,0 +1,588 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800&display=swap');
```

Stylelint (17.14.0)

[error] Unknown rule scss/at-rule-no-unknown. Did you mean at-rule-no-unknown?

(scss/at-rule-no-unknown)